content/guides/models/app/features/panels/media.md

```diff
@@ -0,0 +1,41 @@
+---
+menu:
+  default:
+    identifier: media-panels
+    parent: panels
+title: Media panels
+weight: 50
+---
+
+A media panel visualizes [logged keys for media objects]({{< relref "/guides/models/track/log/media.md" >}}), including 3D objects, audio, images, video, or point clouds. This page shows how to add and manage media panels in a workspace.
+
+{{< img src="/images/app_ui/demo-media-panel.png" alt="Demo of a media panel" >}}
+
+## Add a media panel
+To quick add a media panel for a logged key using the default configuration:
+
+1. To add it globally, click **Add panels** in the control bar near the panel search field.
+1. To add it directly to a section instead, click the section's action `...` menu, then click **Add panels**.
+1. In the list of available panels, find the key for the panel, then click **Add**. Repeat this step for each media panel you want to add, then click the **X** at the top right to close the **Quick Add** list.
+1. Optionally, [configure the panel]({{< relref "#configure-a-media-panel" >}}).
+
+To add a custom media panel manually:
+1. To add it globally, click **Add panels** in the control bar near the panel search field.
+1. To add it directly to a section instead, click the section's action `...` menu, then click **Add panels**.
+1. Click the **Media** section to expand it.
+1. Select the type of media the panel visualizes, 3d objects, images, video, or audio. The panel configuration screen displays. Configure the panel, then click **Apply**. Refer to [Configure a media panel]({{< relref "#configure-a-media-panel" >}}).
+
+## Configure a media panel
+This section shows how to customize a media panel, either when you are manually adding it or afterward. Panels for all media types have the same options.
+
+1. When you a media panel manually, its configuration page opens after you select the type of media. To configure a panel after it is created, click its gear icon.
+1. In the **Basic** tab, give the panel a name, select the keys to visualize, and configure the maximum number of runs, number of columns, and number of items to show.
+1. Optionally configure additional settings in the **Advanced** tab. You can turn on or off **Grid mode** and configure the stride length and snapping behavior for each column.
```

Comment:
```suggestion
1. Optionally configure additional settings in the **Advanced** tab. You can configure smoothing for images, turn on or off **Grid mode** to overlay an X-axis and Y-axis on the panel, adjust the stepper's metric and stride length, and optionally adjust the stepper's offset so that it always stops at the current step.
```
@tristan-spaulding2 PTAL, is this good enough for until I get to the fast follows in https://wandb.atlassian.net/browse/DOCS-1229?